plugins/modules/jenkins_plugin.py

```diff
@@ -632,56 +626,39 @@ def _download_updates(self):
                 msg_exception="Updates download failed.")
 
             # Write the updates file
-            update_fd, updates_file = tempfile.mkstemp()
-            os.write(update_fd, r.read())
+            tmp_update_fd, tmp_updates_file = tempfile.mkstemp()
+            os.write(tmp_update_fd, r.read())
 
             try:
-                os.close(update_fd)
+                os.close(tmp_update_fd)
             except IOError as e:
                 self.module.fail_json(
-                    msg="Cannot close the tmp updates file %s." % updates_file,
+                    msg="Cannot close the tmp updates file %s." % tmp_updates_file,
                     details=to_native(e))
 
         # Open the updates file
         try:
-            f = io.open(updates_file, encoding='utf-8')
+            f = io.open(tmp_updates_file, encoding='utf-8')
+
+            # Read only the second line
+            dummy = f.readline()
+            data = json.loads(f.readline())
```

Comment:
What if there is only one file?

Reply:
Assuming you meant _one line_, that's going to pop an `OSError` that will be captured in the `try`/`except` block in the module.

Reply:
Yes, one line, not file :D